prometheus_client/__init__.py

```diff
@@ -5,18 +5,21 @@
 from . import gc_collector
 from . import platform_collector
 from . import process_collector
+from . import registry
+from . import metrics
+from . import metric_wrappers
 
 __all__ = ['Counter', 'Gauge', 'Summary', 'Histogram', 'Info', 'Enum']
 
-CollectorRegistry = core.CollectorRegistry
-REGISTRY = core.REGISTRY
-Metric = core.Metric
-Counter = core.Counter
-Gauge = core.Gauge
-Summary = core.Summary
-Histogram = core.Histogram
-Info = core.Info
-Enum = core.Enum
+CollectorRegistry = registry.CollectorRegistry
+REGISTRY = registry.REGISTRY
+Metric = metrics.Metric
+Counter = metric_wrappers.Counter
```

Comment:
Users will look for metrics, not metric_wrappers - this naming can be improved.

Reply:
maybe rename `metrics` -> `metrics_core` and `metrics_wrappers` -> `metrics` ?

Reply:
That sounds better. Would you agree with that naming, @brian-brazil?

Reply:
Sure